src/Abstractions/NexusMods.Abstractions.FileStore/Downloads/DownloadAnalysis.cs

```diff
@@ -15,7 +16,7 @@ namespace NexusMods.Abstractions.FileStore.Downloads;
 /// <summary>
 /// Attributes for the analysis data for a downloaded file
 /// </summary>
-public static class DownloadAnalysis
+public partial class DownloadAnalysis : IModelDefinition
```

Comment:
If we keep both yeah. It might be worth making a graph of these and decide what we actually need. 
I'm not sure whether `DownloadAnalysis` still needs to be same entity as the `DownloadTask` .

Reply:
I agree, we should do a pass on the analysis data this sprint